gulpfile.js

```diff
@@ -0,0 +1,33 @@
+const gulp = require('gulp');
+const exec = require('child_process').exec;
+const process = require('process');
+
+gulp.task('start-serverless', function (done) {
+    const instance = exec('node_modules/serverless/bin/serverless offline start &');
```

Comment:
I think this could throw a warning,, we might have to do `node_modules/serverless/bin/serverless.js`
I have seen a few deprecations